src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1709,6 +1831,11 @@ def _ensure_container_insights_for_monitoring(cmd, addon):
     # find the location from the resource group
     location = _get_rg_location(cmd.cli_ctx, resource_group, subscription_id)
 
+    unix_time_in_millis = int(
+        (datetime.datetime.utcnow() - datetime.datetime.utcfromtimestamp(0)).total_seconds() * 1000.0)
```

Comment:
This may be a more concise form:
```
timestamp = (dt - datetime(1970, 1, 1)) / timedelta(seconds=1)
```
From: https://docs.python.org/3/library/datetime.html#datetime.datetime.timestamp